### ISSUE TITLE
Duplicate entries for metadata text fields

### ISSUE THREAD

**Issue by jimdowling** [hopshadoop/hopsworks#178]
I think @evsav meant this behaviour, but for most users it's unexpected.
If you click on edit metadata, then add values. You get multiple metadata values for a text field. 
This might be a new type of field (Array field). But for most use-cases, users would prefer to replace the existing value in the text field with a new value.

**Comment by evsav** [comment]
Yes indeed. I thought this was the behavior we wanted - the db schema is designed accordingly. Having one tuple per table per inode makes things easier though, but reduces our potential (?)
The system also supports multiple templates to inode, thus the total amount of metadata is noOfTemplates x noOfTables x noOfFields x noOfUserFieldInstantiations

**Comment by jimdowling** [comment]
I think we could have a good fix in the UI by adding a "+" button - which means to add an extra value for a field.  So we wouldnt change the backend. All fields could potentially have multiple values. But he UI for "metadata entry" would allow for editing values and adding new values. What do you think?

**Comment by evsav** [comment]
That's it. Metadata entry depending on a user preference
